geopandas/base.py

```diff
@@ -117,6 +117,18 @@ def area(self):
         3     0.0
         4     0.0
         dtype: float64
+
+        See also
+        --------
+        GeoSeries.length : measure length
+
+        Notes
+        -----
+        Area may be invalid for a geographic CRS; use ``to_crs`` to project
```

Comment:
maybe explain more the *why* here as well? Or be more explicit like "geographic CRS using degrees unit"